File: Leetcode 3774. Absolute difference between max and min k elements/Solution.py
class Solution:
    def absDifference(self, nums: List[int], k: int) -> int:
        nums.sort()
        s1 = s2 = 0
        for i in range(k):
            s1 += nums[i]
            s2 += nums[len(nums) - 1 - i]
        return abs(s1 - s2)

File: Leetcode 3774. Absolute difference between max and min k elements/Solution.java
import java.util.Arrays;

class Solution {
    public int absDifference(int[] nums, int k) {
        Arrays.sort(nums);
        int max_sum=0, min_sum=0;
        int n=nums.length;

        for(int i=0; i<k; i++) {
            min_sum += nums[i];
            max_sum += nums[n-i-1];
        }
        return Math.abs(max_sum - min_sum);
    }
}
